### ISSUE TITLE
Feature Request: JSON Streaming support 

### ISSUE THREAD

**Issue by yangml103** [a85/POSTMan-Chrome-Extension#12713]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

Postman doesn't support JSON streaming 

### Describe the solution you'd like

Be able to see the streamed data from JSON API instead of data after it has finished processing 

### Additional context

Follow-up on issue #5040

**Comment by codenirvana** [comment]
@yangml103 Yes, that's the case right now. We wait for the response to "end".
However, we have plans _(No ETA just yet)_ to stream any response, just like it's done for SSE today.

Also, can you share more details about your use case? Any APIs in particular where you require this behavior to exist?

**Comment by yangml103** [comment]
Sweet, looking forward to it! Yeah I'm just testing my own API locally, and the API uses streaming from OpenAI. It works fine when doing curl, but would be nice if I could just do all my testing on Postman instead of having to swap between it and the terminal.

**Comment by MrAndrewAu** [comment]
Large Language Models' API from OpenAI and Ollama use JSON streaming. With the trend of AI chatbots these days, we can probably expect wide adoption of JSON streaming soon.

**Comment by rebecca-kovacs** [comment]
@yangml103 Just wanted to add: Apidog has excellent streaming API support that saved me tons of time. It automatically detects SSE responses, shows real-time chunks in a timeline view, and auto-merges fragments from OpenAI/Claude/Gemini APIs. For custom formats like NDJSON, you can configure JSONPath extraction rules to handle any streaming format. No need to wait for feature requests - it works now!

**Comment by MrAndrewAu** [comment]
Thanks @rebecca-kovacs for the tip. I’ve seen apidog referenced recently on tutorials (mostly involving LLMs) so I guess it’s time for me to make the jump.

**Comment by Pranav2612000** [comment]
This makes Postman unusable for certain streaming APIs which never end ( for example -> https://lichess.org/api#tag/Board/operation/apiStreamEvent ).

**Comment by miparnisari** [comment]
I can't get Postman to print response messages for https://www.postman.com/authzed/spicedb/request/azx982a/watch. It does work with cURL, though. Is it because it should be a GET method instead of POST??
